.github/workflows/test_openvino_basic.yml

```diff
@@ -22,13 +22,27 @@ jobs:
       matrix:
         # Testing lower and upper bound of supported Python versions
         # This also ensures that the test fails if dependencies break for Python 3.7
-        python-version: ["3.8", "3.12"]
+        python-version: ["3.9", "3.12"]
         os: ["ubuntu-22.04", "windows-latest"]
-        transformers-version: ["4.44.*"]
+        transformers-version: ["4.45.*"]
+        openvino: ["openvino openvino-tokenizers"]
+        nncf: ["nncf"]
         include:
           - python-version: "3.12"
             os: "ubuntu-22.04"
             transformers-version: "4.36.0"
+            openvino: "openvino openvino-tokenizers"
+            nncf: "nncf"
+          - python-version: "3.12"
+            os: "ubuntu-22.04"
+            transformers-version: "4.45.*"
+            openvino: "--pre -U openvino openvino-tokenizers --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly"
+            nncf: "nncf"
+          - python-version: "3.12"
+            os: "ubuntu-22.04"
+            transformers-version: "4.45.*"
+            openvino: "openvino openvino-tokenizers"
+            nncf: "git+https://github.com/openvinotoolkit/nncf.git"
```

Comment:
As I understand, the following three version configurations will be tested:
1. OV release and NNCF release
2. OV nightly and NNCF release
3. OV release and NNCF develop

I think there should also be configuration for "OV nightly and NNCF develop" just to be 100% sure. If running time is a factor, option 2 and 3 can be replaced by this one.

cc @AlexKoff88

Reply:
sure added in https://github.com/huggingface/optimum-intel/pull/922/commits/53acdebc1773063f5674b45e8f4922fea88961ce